Helm keycloak updates

karun-singh opened the pull request:
- Fixed secret creation commands.
- Updated add repo command with appropriate commit URL for the required index containing the keycloak chart version used in IUDX.

abhi4578 reviewed (approved):
LGTM

abhi4578 reviewed (changes requested):
seperate resource values (memory, cpu, storageclass, node-selector) for elastic, kibana and logstash for azure and aws.

karun-singh commented:
- Created example resource value yamls for aws and azure. 
- Renamed keycloak-values.yaml to values.yaml to make the naming consistent with other components.
- Updated README
